src/main/java/io/xpring/xrpl/DefaultXpringClient.java

```diff
@@ -112,6 +121,14 @@ public int getLatestValidatedLedgerSequence() throws XpringKitException {
 
     @Override
     public RawTransactionStatus getRawTransactionStatus(String transactionHash) throws XpringKitException {
-        throw XpringKitException.unimplemented;
+        Objects.requireNonNull(transactionHash);
```

Comment:
non-blocking: Why did you choose to have the `requireNonNull` check in this method rather than `TransactionStatus` above?

Reply:
This is an oversight. Added.